README.md

```diff
@@ -1,11 +1,11 @@
-# Nexmo Rails Gem
+# Vonage Rails Gem
 
-[![Gem Version](https://badge.fury.io/rb/nexmo_rails.svg)](https://badge.fury.io/rb/nexmo_rails) [![Coverage Status](https://coveralls.io/repos/github/Nexmo/nexmo-rails/badge.svg?branch=master)](https://coveralls.io/github/Nexmo/nexmo-rails?branch=master) [![Build Status](https://api.travis-ci.org/Nexmo/nexmo-rails.svg?branch=master)](https://travis-ci.org/Nexmo/nexmo-ruby)
+[![Gem Version](https://badge.fury.io/rb/vonage_rails.svg)](https://badge.fury.io/rb/vonage_rails) [![Coverage Status](https://coveralls.io/repos/github/Nexmo/nexmo-rails/badge.svg?branch=master)](https://coveralls.io/github/Nexmo/nexmo-rails?branch=master) [![Build Status](https://api.travis-ci.org/Nexmo/nexmo-rails.svg?branch=master)](https://travis-ci.org/Nexmo/nexmo-rails)
```

Comment:
Coveralls will need moved

Reply:
We aren't moving the repo yet, but when we do this will need to be moved 100%